docs/conf.py

```diff
@@ -349,3 +349,5 @@
 
 # -------------------------------------------------------------------------
 # nitpicky = True
+
+nitpick_ignore = [("py:mod", "aiohttp")]
```

Comment:
Let's leave comments next to each ignore so that in the future we'd know how to fix this:
```suggestion
nitpick_ignore = [
    ("py:mod", "aiohttp"),  # undocumented, no `.. currentmodule:: aiohttp` in docs
]
```